addons/antialiased_line2d/antialiased_line2d.gd

```diff
@@ -6,3 +6,14 @@ extends Line2D
 func _ready() -> void:
 	texture = AntialiasedLine2DTexture.texture
 	texture_mode = Line2D.LINE_TEXTURE_TILE
+
+
+static func construct_closed_line(p_polygon: PoolVector2Array) -> PoolVector2Array:
+	var end_point: Vector2 = p_polygon[p_polygon.size() - 1]
+	var distance: float = end_point.distance_to(p_polygon[0]) # distance to start point
+	var bridge_point: Vector2 = end_point.move_toward(p_polygon[0], distance / 2.0)
```

Comment:
Use multiplication instead of division as it's (slightly) faster:

```suggestion
	var bridge_point: Vector2 = end_point.move_toward(p_polygon[0], distance * 0.5)
```